charts/stable/gollum/Chart.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: v2
+appVersion: latest
+description: Gollum is a simple wiki system built on top of Git
+name: gollum
+version: 1.0.0
+kubeVersion: ">=1.16.0-0"
+keywords:
+- gollum
+- wiki
+home: https://github.com/k8s-at-home/charts/tree/master/charts/stable/gollum
+icon: https://gollum.org/icon
+sources:
+- https://github.com/gollum/gollum
+- https://github.com/gollum/docker
+maintainers:
+- name: Chris Sanders
```

```suggestion
- name: @chris-sanders
```
`ct` apparently wants to have an actual GitHub id here